Omar Rosad Ramirez
801-12-8150
CCOM3986
Prof. H. Ortiz-Zuazaga

Lab Notes : week 8 Mar4-Mar8

**SUMMARY**

This is my second week in the lab. Still digesting the paper and setting up
the development environment for kevlar on my laptop. The setup is giving me
ideas on what to include in the README, considering some errors that have surfaced.



**DEFINITIONS**

Single Nucleotide Variant(SNV): Means theres a difference in one base. Example: ATG -> CTG, C and A differ ;
                                This definition **does not imply** any kind of how often this variation occurs, just that there is at
                                this particular nucleotide a difference.

Single Nucleotide Polymorphism(SNP): These **do imply** that there is an implication how often this variation occurs. 


**MISC COPY/PASTE**

Any algorithm must be confident not only in the existence of the variant in the proband but also its non-existence in both parents ;

 In a reference-mapping context, calling indels with confidence requires accurate mapping of each read spanning the indel, with all gaps arranged consistently.  This is possible only for short indels,  and tends to be prone to error and misalignment. Thus, prediction of indels with length > 10 bp has proven to be very challenging and accompasnied by high false positive and false negative rates.

  Furthermore, the prediction of SVs is only possible through indirect signatures such as alterations in read depth or read-pair signatures. These signatures can be quite noisy and result in high rate of false negative and false positive prediction. As a result,some basic properties of de novo SVs, including their rate of occurrence, remain unknown. It is important to note that there also exists no method for predicting of more complex types of de novo SVs, such as inversion-duplication.

**QUESTIONS** 

